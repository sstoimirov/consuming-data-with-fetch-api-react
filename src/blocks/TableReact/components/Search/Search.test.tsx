import * as React from "react";
import { shallow, configure } from "enzyme";
import { Search } from "./Search";
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

describe("Search", () => {
    it("Renders correctly", () => {
        const comp = shallow(<Search text="Place you text here" onChange={x => x} />);
        expect(comp.debug()).toMatchSnapshot();
    })
})