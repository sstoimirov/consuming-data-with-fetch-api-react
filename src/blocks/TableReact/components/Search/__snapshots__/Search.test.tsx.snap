// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Renders correctly 1`] = `"<input type=\\"text\\" placeholder=\\"Place you text here\\" onChange={[Function: onChange]} />"`;
